.github/workflows/mavenCI.yaml

```
@@ -0,0 +1,28 @@
+name: Java CI with Maven
+
+on:
+  push:
+    branches: [ "main" ]
```

I changed the default branch to "develop" so i would add this to this list
```suggestion
    branches: [ "main", "develop" ]
```